Selenium.Axe/Selenium.Axe.Test/integration-test-target-complex.html

```diff
@@ -0,0 +1,1456 @@
+<!DOCTYPE html>
```

Comment:
I think this integration test file has a few issues:

1. It includes code from another project but not the license/copyright header from that other project
2. It relies on several not-included javascript files for functionality; as-is, it just loads as a blank page with a bunch of errors in the console, and without those other files, it doesn't actually seem to reproduce the issue it's meant to act as a pinning test for

I think the best way to solve both 1 and 2 would be to use a more minimal repro case, rather than checking in this complex case. Per [axe-core's API docs](https://www.deque.com/axe/core-documentation/api-documentation/#results-object), the thing that triggers the complex `Target` property is (**emphasis** mine):

> `target` - Array of either strings or Arrays of strings. If the item in the array is a string, then it is a CSS selector. If there are multiple items in the array each item corresponds to one level of iframe or frame. If there is one iframe or frame, there should be two entries in target. If there are three iframe levels, there should be four entries in target. **If the item in the Array is an Array of strings, then it points to an element in a shadow DOM and each item (except the n-1th) in this array is a selector to a DOM element with a shadow DOM.** The last element in the array points to the final shadow DOM node.

So I think all we should need to create a repro case for this bug is a page that contains an iframe, which in turn contains a shadow root, which in turn contains some axe-core violation. Here's an example of a self-contained minimal repro page:

```html
<div id="container"></div>
<script>
    let container = document.getElementById('container');
    if (window.top === window.self) {
        let iframe = document.createElement('iframe');
        iframe.src = window.location.href
        container.appendChild(iframe);
    } else {
        let shadowRoot = container.attachShadow({mode: 'open'});
        shadowRoot.innerHTML = '<p style="color: #aaaaaa">low-contrast</p>';
        shadowRoot.appendChild(lowContrastText);
    }
</script>
```

Reply:
Excellent stuff @dbjorge  - thanks for isolating the issue!

I have updated the test case based on this.